src/player.cpp

```diff
@@ -1334,21 +1338,14 @@ int player::floor_bedding_warmth( const tripoint &pos )
     int floor_bedding_warmth = 0;
 
     const optional_vpart_position vp = g->m.veh_at( pos );
-    const bool veh_bed = static_cast<bool>( vp.part_with_feature( "BED", true ) );
-    const bool veh_seat = static_cast<bool>( vp.part_with_feature( "SEAT", true ) );
 
     // Search the floor for bedding
     if( furn_at_pos != f_null ) {
         floor_bedding_warmth += furn_at_pos.obj().floor_bedding_warmth;
     } else if( !trap_at_pos.is_null() ) {
         floor_bedding_warmth += trap_at_pos.floor_bedding_warmth;
-    } else if( veh_bed && veh_seat ) {
-        // BED+SEAT is intentionally worse than just BED
-        floor_bedding_warmth += 250;
-    } else if( veh_bed ) {
-        floor_bedding_warmth += 300;
-    } else if( veh_seat ) {
-        floor_bedding_warmth += 200;
+    } else if( vp.part_with_feature( "BOARDABLE", true ) ) {
```

Comment:
```suggestion
    } else if( auto boardable = vp.part_with_feature( "BOARDABLE", true ) ) {
        floor_bedding_warmth += boardable ->info().floor_bedding_warmth;
```
No need to call `part_with_feature` again to get exactly the same value from it.